website/docs/04_comparisons/02_timoni.md

```diff
@@ -0,0 +1,20 @@
+---
+slug: timoni
+sidebar_label: With Timoni
+title: Glasskube vs Timoni Comparison
+description: Timoni installs and updates cloud-native applications as bundles (OCI images) with a great support for configuration, but it requires users to create its own bundles.
+---
+
+# Glasskube vs Timoni Comparison
+
+[Timoni](https://helm.sh/) was created by [Stefan Prodman](https://github.com/stefanprodan) to improve bundling cloud native applications and their lifecycle management.
+
+Timoni is closer related to helm than Glasskube, but brings some features and new approaches to the table Helm doesn't offer.
+Its bundles are OCI images, so the only way to install these bundles is to publish them first to an OCI registry, which makes additional security features like image (co-)singing possible. 
+Similar to Glasskube it performs garbage collection of orphan resources after an uninstallation operation.
+
+Timoni is in general already more mature than Glasskube, but doesn't focus on providing an easy way of package installation and upgrades and as of now there is also no GUI support planned
```

Comment:
I think this sentence is too long and has too many "and"s. I think it should be split into two sentences.